isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -4350,26 +4316,26 @@ namespace isobus
 			return false;
 		}
 
-		if (replace && replace_command(data, priority))
+		if (replace && replace_command(data))
 		{
 			return true;
 		}
 
-		commandQueue.emplace_back(data, priority);
+		commandQueue.emplace_back(data);
 		return true;
 	}
 
-	bool VirtualTerminalClient::replace_command(const std::vector<std::uint8_t> &data, CANIdentifier::CANPriority priority)
+	bool VirtualTerminalClient::replace_command(const std::vector<std::uint8_t> &data)
 	{
 		bool alreadyReplaced = false;
 		for (auto it = commandQueue.begin(); it != commandQueue.end();)
 		{
-			bool matchesFunctionCode = (it->first[0] == data[0]);
-			if (matchesFunctionCode && (it->second == priority))
+			bool matchesFunctionCode = (it[0][0] == data[0]);
+			if (matchesFunctionCode)
 			{
 				if (!alreadyReplaced)
 				{
-					it->first = data;
+					it[0] = data;
```

Comment:
```suggestion
					*it = data;
```
Same here, not sure if there's a difference. But I feel like the `*` notation makes it easier to gasp that we're still talking about a complete vector of data here